guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx

```diff
@@ -101,7 +101,7 @@ export class CommonControlPropertyGridComponent extends React.Component<ICommonC
                     <TextInputLineComponent lockObject={this.props.lockObject} label="Y" target={control} propertyName="top" onPropertyChangedObservable={this.props.onPropertyChangedObservable} />
                 </div>
                 <div className="divider">
-                    <TextInputLineComponent icon={sizeIcon} lockObject={this.props.lockObject} label="W" target={control} propertyName="width" onPropertyChangedObservable={this.props.onPropertyChangedObservable}
+                    <TextInputLineComponent iconLabel={"Sca;e"} icon={sizeIcon} lockObject={this.props.lockObject} label="W" target={control} propertyName="width" onPropertyChangedObservable={this.props.onPropertyChangedObservable}

```

Comment:
is iconLabel = {"Sca;e"} correct? The semicolon stands out as a potential typo

Reply:
AHA. Whoops. ;)